.eslintrc

```diff
@@ -55,7 +54,7 @@
     "no-negated-in-lhs": 2,
     "no-obj-calls": 2,
     "no-regex-spaces": 2,
-    "quote-props": [2, "as-needed", {"keywords": true}],
+    "quote-props": [2, "as-needed", {"keywords": true, "unnecessary": false}],
```

Comment:
Without that, the current code base throws 40 "Unnecessarily quoted property" errors.
For example, in regressions.js:

    4:7   error  Unnecessarily quoted property 'books' found        quote-props
    5:9   error  Unnecessarily quoted property 'title' found        quote-props
    6:9   error  Unnecessarily quoted property 'author' found       quote-props
    7:11  error  Unnecessarily quoted property 'name' found         quote-props